functions/controllers/user/imageUpload.js

```diff
@@ -0,0 +1,54 @@
+const Busboy = require('busboy')
+const path = require('path')
+const os = require('os');
+const fs = require('fs')
+const config = require('../../util/config')
+const { admin } = require('../../util/admin')
+const { status, message } = require("../../util/constants");
+var HttpStatus = require("http-status-codes");
+
+config.storageBucket
+
+const imageUpload = async (req, res, db) => {
+    const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = HttpStatus
+    const { error, success } = status
+    const { wrongFileSubmitted, somethingWentWrong, imageUpdateSucces } = message
+
+    const busboy = new Busboy({headers: req.headers});
+    let imageFileName
+    let imageToBeUploaded = {};
+
+    await busboy.on('file', (fieldname, file, filename, encoding, mimetype) => {
+        if(mimetype !== 'image/jpeg' && mimetype !== 'image/png'){
```

Comment:
Can you use constants for 'image/jpeg'and 'image/png'
```suggestion
        if(mimetype !== IMAGE_JPEG && mimetype !== IMAGE_PNG){
```